### PR TITLE
Issue 7 re-write test suites adddelete tests as needed

### PR DESCRIPTION
**Pull request by IslaL**
Rewrite test suites and test util files as needed.

### CODE REVIEW

File: onc/tests/globals.m [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
-addpath(genpath('util'));
-addpath(genpath('suites'));
-addpath(genpath('../'));
+    % grab token from "TOKEN" file
+    f = fopen('TOKEN','r');
+    if f > 0
+       line = fgetl(f);
+       fclose(f);
+    else
+       line = getenv('TOKEN_STRING');
```
Comment (KristenMeyerONC) [inline onc/tests/globals.m]: Could we give this a more descriptive name? token perhaps?

**Review by KristenMeyerONC — APPROVED** [review]

File: onc/tests/globals.m (hunk 1 above)
Comment (IslaL) [reply onc/tests/globals.m]: Yes! The user's token file is named token too, maybe a bit confusing. How about initialize.m or getConfig.m or some others?

Comment (KristenMeyerONC) [reply onc/tests/globals.m]: I think the function names are ok. I was meaning the variable "line"

Comment (IslaL) [reply onc/tests/globals.m]: ohhh I see. I will update that to token.

**Pull request merged**